### PR TITLE
[BugFix] use be/cn when publish version  for cloudnative table

### PR DESCRIPTION
**Pull request by abc982627271**
Fixes #issue

## What type of PR is this:
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 4 / 14 (28.57%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/Utils.java|4|14|28.57%|[105, 155, 162, 164, 165, 166, 174, 177, 184, 188]|

File: fe/fe-core/src/main/java/com/starrocks/lake/Utils.java [hunk 1]
```diff
@@ -152,38 +152,40 @@ public static void publishLogVersion(@NotNull List<Tablet> tablets, long txnId,
         for (Tablet tablet : tablets) {
             Long beId = Utils.chooseBackend((LakeTablet) tablet);
```
[the remark this answers is not in the document]
Comment (abc982627271) [reply fe/fe-core/src/main/java/com/starrocks/lake/Utils.java:153]: It won't

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 4 / 14 (28.57%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/Utils.java|4|14|28.57%|[105, 155, 162, 164, 165, 166, 174, 177, 184, 188]|

**Review by kevincai — APPROVED** [review]

**Review by wyb — APPROVED** [review]

**Comment by andyziye** [comment]
@Mergifyio rebase

**Review by sduzh — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 4 / 14 (28.57%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/Utils.java|4|14|28.57%|[105, 155, 162, 164, 165, 166, 174, 177, 184, 188]|

**Pull request merged**